src/javascript/app/pages/bottom/tabs.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { SegmentedControlSingleChoice, Skeleton } from '@deriv-com/quill-ui';
+import { Explanation } from './explanation.jsx';
+import { getElementById } from '../../../_common/common_functions';
+import WebtraderChart from '../trade/charts/webtrader_chart';
+import { useMarketChange, useContractChange } from '../../hooks/events';
+import { localize } from '../../../_common/localize';
+import dataManager from '../../common/data_manager.js';
+import LastDigit from '../trade/last_digit.jsx';
+
+const Graph = ({ renderGraph }) => {
+    const [isMounted,setMounted] = useState(false);
+    useEffect(() => {
+        setTimeout(() => {
+            setMounted(true);
+            setTimeout(() => {
+                renderGraph();
+            }, 1000);
+        }, 1000);
+    },[]);
+
+    if (isMounted){
+        return (
+            <div id='tab_graph' className='chart-section'>
+                <p className='error-msg' id='chart-error' />
+                <div id='trade_live_chart'>
+                    <div id='webtrader_chart' />
+                </div>
+            </div>
+        );
+    }
+
+    return <Skeleton.Square rounded fullWidth height={300} />;
+};
+
+const BottomTabs = () => {
+    const hasMarketChange = useMarketChange();
+    const [selectedTab, setSelectedTab] = useState(1);
+    const [hasLastDigit, setHasLastDigit] = useState(false);
+    const [formName, setFormName] = useState('');
+    const hasContractChange = useContractChange();
+    const savedTab = sessionStorage.getItem('currentTab');
+
+    const renderGraph = (callback) => {
+        setTimeout(() => {
+            WebtraderChart.cleanupChart();
+            WebtraderChart.showChart();
+
+            if (typeof callback === 'function') {
```

Comment:
If type of callback can be `undefined` or `function`, then will `callback?.()` work?